Update EIP overview pages (#738)
wso2/docs-mi: en/docs/learn/enterprise-integration-patterns/system-management/overview.md

> System Management patterns aim to monitor the amount of sent messages and their processing times without analyzing the message data, except for some fields in the message header in selected patterns. This is in contrast to Business Activity Monitoring, which directly analyzes the payload data contained in a message.

Reviewer: ```suggestion
System Management patterns aim to monitor the number of messages sent and their processing times without analyzing the message data, except for some fields in the message header in selected patterns. This is in contrast to Business Activity Monitoring, which directly analyzes the payload data contained in a message.
```

> This chapter introduces various system management patterns and how each can be simulated using the WSO2 Micro Integrator.

Reviewer: ```suggestion
This section introduces various system management patterns and how they can be simulated using the WSO2 Micro Integrator.
```
